queries.sql

```diff
@@ -1,3 +1,44 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
\ No newline at end of file
+SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth < '2019-01-01';
+SELECT name FROM animals WHERE neutered = 't' AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR 'Pikachu';
+SELECT  name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * from animals WHERE neutered = 'true';
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg > 10.4 AND weight_kg =< 17.3;
+
+/* task-2 */
+BEGIN;
+   UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+   UPDATE animals SET species = 'pokemon' WHERE species = null;
+COMMIT;
```

Comment:
* Kindly Verify that change was made and persists after commit. as stated in the project requirement [Check here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made%20and%20persists%20after%20commit.)

Reply:
:heavy_check_mark: Done!